api/posts/postsSchemas.js

```diff
@@ -16,13 +21,37 @@ const postSchemaCommonKeys = {
 
   viewsCount: Joi.number().integer().min(0).default(0),
 
-  imageUrl: Joi.string().regex(URL_REGEXP).default("")
+  imageUrl: Joi.string().regex(URL_REGEXP).default(""),
+
+  isDeleted: Joi.boolean().default(false)
 };
 
 const newPostSchema = {
   body: Joi.object().keys(Object.assign({}, postSchemaCommonKeys))
 };
 
+const allPostsSchema = {
+  query: Joi.object().keys({
+    page: Joi.string().alphanum().trim().default("1"),
+    perPage: Joi.number().integer().min(1).max(100).default(3),
+    sortBy: Joi.string()
+      .trim()
+      .regex(SORT_BY)
+      .default("asc(_id)")
+      .error(
+        new Error(
+          "Please enter a valid 'sortBy' parameter. Must be asc(parametr to sort) or desc(parametr to sort)"
+        )
+      ),
+    titleEq: Joi.string().trim(),
+    dateGte: Joi.string().trim().regex(DATE),
+    dateLte: Joi.string().trim().regex(DATE),
+    tagEq: Joi.string().trim(),
```

Comment:
I would make it like just a `tag`, there is no needs to add `Eq` because it is obvious eq. 
And it could add some additional work it the future

Reply:
agree! I've fixed